otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1517,5 +1517,19 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('ssi-dim-wallet-stub') {
+      description: "Stub application for DIM Wallet",
+      allow_merge_commit: true,
+      dependabot_alerts_enabled: true,
+      dependabot_security_updates_enabled: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      has_discussions: true,
+      private_vulnerability_reporting_enabled: true,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
please use "read" default permissions for new repos.

Reply:
Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/4c77d7ab506deac405bb300e0a3dbbef4a92574c